dmscripts/send_dos_opportunities_email.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime, date, timedelta
+
+from requests.exceptions import RequestException
+
+from dmscripts.helpers.html_helpers import render_html
+from dmscripts.helpers import logging_helpers
+from dmscripts.helpers.logging_helpers import logging
+from dmutils.formats import DATETIME_FORMAT
+
+import dmapiclient
+
+logger = logging_helpers.configure_logger({'dmapiclient': logging.INFO})
+
+
+def get_live_briefs_between_two_dates(data_api_client, lot_slug, start_date, end_date):
+    return [
+        brief for brief in data_api_client.find_briefs_iter(status="live", lot=lot_slug)
+        if datetime.strptime(brief['publishedAt'], DATETIME_FORMAT).date() >= start_date
+        and datetime.strptime(brief['publishedAt'], DATETIME_FORMAT).date() <= end_date
+    ]
+
+
+def get_campaign_data(lot_name, list_id):
+    return {
+        "type": "regular",
+        "recipients": {
+            "list_id": list_id,
+        },
+        "settings": {
+            "subject_line": "New opportunities for {0}: Digital Outcomes and Specialists 2".format(lot_name),
+            "title": "DOS Suppliers: {0} [{1}]".format(lot_name, datetime.now().strftime("%d %B")),
+            "from_name": "Digital Marketplace Team",
+            "reply_to": "do-not-reply@digitalmarketplace.service.gov.uk",
+            "use_conversation": False,
+            "authenticate": True,
+            "auto_footer": False,
+            "inline_css": False,
+            "auto_tweet": False,
+            "fb_comments": False
+        },
+        "tracking": {
+            "opens": True,
+            "html_clicks": True,
+            "text_clicks": False,
+            "goal_tracking": False,
+            "ecomm360": False
+        }
+    }
+
+
+def get_html_content():
+    # function not yet implemented
+    html_content = render_html("email_templates/dos_opportunities.html", data={})
+    return {"html": html_content}
+
+
+def create_campaign(mailchimp_client, campaign_data):
```

Comment:
Just a note: `create_campaign`, `set_campaign_content` and `send_campaign` will implicitly return `None` if no return line is specified. This can be used in a truthey/ falsey check as None is Falsey.
_However_ 'explicit is better than implicit' and I think in this case explicitly defining the returns makes the code more readable so I'm happy for them to stay in.